src/components/macroview/rightPanel/EditArea.tsx

```diff
@@ -9,6 +9,36 @@ import KeyPressForm from './editForms/KeyPressForm'
 import MousePressForm from './editForms/MousePressForm'
 import SystemEventActionForm from './editForms/SystemEventActionForm'
 
+export function BoxText({ text }: { text: string }) {
```

Comment:
Seems kinda strange to have this component in this file when it's not used in this file

Reply:
You should also define the `text` bit as `children`, so it can be used as `<BoxText>I am the text</BoxText>`

Reply:
I added it there because its used from all the underlying editForms and this seemed like a logical place to put it above each of the forms that use it.

I tweaked the logic itself: [155e6b4](https://github.com/WootingKb/wooting-macros/pull/194/commits/155e6b49af235ca04893e226dffc6dc22f06df8d)